model_compression_toolkit/core/common/quantization/quantization_params_generation/qparams_activations_computation.py

```diff
@@ -66,7 +66,7 @@ def get_activations_qparams(activation_quant_cfg: NodeActivationQuantizationConf
                 activation_quant_cfg.activation_quantization_params_fn = \
                     quantization_params_generation.uniform_no_clipping_selection_min_max
     else:
-        signed = np.any(bins_values[:-1][bins_counts > 0] < 0)
+        signed = np.any(bins_values[:-1][bins_counts > 0] < 0)  # pragma: no cover
```

Comment:
Why no pragma?

Reply:
The line execute when prior info hasn't been set.
It's not possible while using the quantization facade, but I'll do a function test.

Reply:
If this case is not feasible then make it an error otherwise test it